applications_tests/gls_navier_stokes_2d/mms2d_cahn_hilliard.prm

```diff
@@ -0,0 +1,161 @@
+# Listing of Parameters
+# ---------------------
+# Simulation and IO Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method            = steady
+  set number mesh adapt = 3
+  set output frequency  = 0
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+
+subsection multiphysics
+  set fluid dynamics = true
+  set cahn hilliard  = true
+end
+
+#---------------------------------------------------
+# Cahn-Hilliard order
+#---------------------------------------------------
+
+subsection FEM
+  set phase ch order     = 1
+  set potential ch order = 1
+  set velocity order     = 1
+  set pressure order     = 1
+end
+
+#---------------------------------------------------
+# Analytical solution
+#---------------------------------------------------
+
+subsection analytical solution
+  set enable    = true
+  set verbosity = verbose
+
+  subsection cahn hilliard
+    set Function expression = sin(pi*x)*sin(pi*y);0
```

Comment:
Yes, we are only performing a mms on the phase field variable since in the end, we're only interested in the phase-field. It is very hard to impose a source term both on potential and phase-field using the Cahn-Hilliard equations.

Reply:
edit : the chemical potential analytical solution was set to 1/0.0 in order for the terminal to display inf for all configuration thus making it less fragile than currently